### ISSUE TITLE
Attributes aria-posinset and aria-setsize not allowed on element a at this point

### ISSUE THREAD

**Issue by Ignacio-de-Loyola** [validator/validator#1540]
This is code to reproduce error.
`<h3 class="_hidden" id="offers-feed">Ongoning special Offers</h3>
<section role="feed" aria-labelledby="offers-feed">
  <ul>
    <li><a role="article" aria-posinset="1" aria-setsize="-1"
     aria-labelledby="article-title-1" aria-describedby="article-period-1" tabindex="0">
       <p id="article-title-1">Event 1</p>
       <p id="article-period-1">Event period</p>
    </a></li>
    <li><a role="article" aria-posinset="2" aria-setsize="-1"
     aria-labelledby="article-title-2" aria-describedby="article-period-2" tabindex="0">
       <p id="article-title-2">Event 2</p>
       <p id="article-period-2">Event period</p>
    </a></li>
  </ul>
</section>`

[#1060](https://github.com/validator/validator/issues/1060) You said fixed this, but still error.
Please can you confirm, this code is w3c compliant?

Thanks,

**Comment by scottaohara** [comment]
these attributes should be allowed on an element which allows the article role to be set, or the article element itself. so that does seem something that needs to be fixed.  However, what's not mentioned in the error output is that you should not be using both the aria attributes on the 'article' and the list elements.

a feed is supposed to directly contain articles, and the use of aria-posinset and aria-setsize are meant to communicate the current number and total number (if known) of those articles.

by placing these articles within a list, you're adding extra semantics that could be announced to screen readers, and the numeration provided by the list items would likely either be repetitive or conflict to the specific numeration that you're adding to the article elements.

**Comment by Ignacio-de-Loyola** [comment]
This is W3C aria-setsize example.
![image](https://user-images.githubusercontent.com/80868256/218038947-1aa8bc2f-8b4f-4184-85d0-69ad9c313aa1.png)

These are using both the aria attributes on the 'option' and the list elements.
Is is also problem?

**Comment by Ignacio-de-Loyola** [comment]
These example using both the aria attributes(aria-posinset and aria-setsize) on the 'option' and the list elements.

![image](https://user-images.githubusercontent.com/80868256/218040744-e3e246a4-bb16-411a-8262-dffd3e0e0c24.png)
[W3C example](https://www.w3.org/TR/wai-aria/#aria-setsize)
Is it also error?

**Comment by scottaohara** [comment]
no, and the information that describes that example explains why.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/1540#issuecomment-1425717532
